lucien-ios/ProfileViewController.swift

```diff
@@ -16,18 +16,61 @@ final class ProfileViewController: UIViewController {
     @IBOutlet private weak var name: UILabel!
     @IBOutlet private weak var emailTitle: UILabel!
     @IBOutlet private weak var email: UILabel!
-    @IBOutlet private weak var profileTitle: UILabel!
+
+    let loginViewModel = LoginViewModel()
 
     override func viewDidLoad() {
         super.viewDidLoad()
+        configureNavigationController()
+        loginViewModel.getCurrentUser {
+            guard let currentUser = self.loginViewModel.currentUser else { return }
+            debugPrint("in debug print \(currentUser)")
+            self.name.text = currentUser.firstName + " " + currentUser.lastName
+            self.email.text = currentUser.email
+        }
+        setUpStyling()
+    }
+
+    @objc func back(sender: UIBarButtonItem) {
+        self.navigationController?.popViewController(animated: true)
+    }
+
+    private func configureNavigationController() {
+        guard self.navigationController != nil else { return }
+        UINavigationBar.setNavBarBackground(navigationController: self.navigationController!)
+        self.navigationItem.title = "Profile"
+        UINavigationBar.setNavBarTitle(navigationController: self.navigationController!, title: "Profile")
+        setNavBarBackButton()
+    }
+
+    private func setNavBarBackButton() {
+        let backButton = UIBarButtonItem(image: UIImage(named: "navBackButton"), style: .plain, target: self, action: #selector(back(sender:)))
+        backButton.tintColor = LucienTheme.dark
+        self.navigationItem.leftBarButtonItem = backButton
+    }
+
+    private func setUpStyling() {
         emptyProfilePicture.contentMode = .scaleAspectFit
         logoutButton.backgroundColor = LucienTheme.dark
+        logoutButton.titleLabel?.font = LucienTheme.Fonts.muliBold(size: 13.0)
         logoutButton.setTitleColor(UIColor.white, for: .normal)
         logoutButton.layer.cornerRadius = 3.0
         nameTitle.textColor = LucienTheme.coolGrey
         emailTitle.textColor = LucienTheme.coolGrey
         name.textColor = LucienTheme.dark
         email.textColor = LucienTheme.dark
-        profileTitle.textColor = LucienTheme.dark
+        nameTitle.font = LucienTheme.Fonts.muliLight(size: 12.0)
+        emailTitle.font = LucienTheme.Fonts.muliLight(size: 12.0)
+        name.font = LucienTheme.Fonts.muliRegular(size: 18.0)
+        email.font = LucienTheme.Fonts.muliRegular(size: 18.0)
+
+
+
+
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Vertical Whitespace Violation: Limit vertical whitespace to a single empty line. Currently 4. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.intrepid.io/coding_rules#rule_key=SwiftLint%3Avertical_whitespace)